github_pr.py

```diff
@@ -389,9 +431,9 @@ def main():
     parser.add_argument('--noheaders', action='store_true', help='remove headers from table view. best for programmatic use of this script')
     parser.add_argument('--noratelimit', action='store_true', help="don't show the rate limit")
     parser.add_argument('--mergecomment', default=":shipit:", help='string to look for when checking comments for "shipit" approval, during MERGE only')
-    parser.add_argument('--mergecondition-owner-cannot-ship', action='store_true', help='stops owner from being able to apply merge comment')
-    parser.add_argument('--mergecondition-approved-mergers', default=None, nargs='*', help='list of usernames of approved mergers')
-    parser.add_argument('--mergecondition-approved-mergers-file', default=None, help='file of usernames of approved mergers')
+    parser.add_argument('--condition-owner-cannot-ship', action='store_true', help='stops owner from being able to apply merge comment')
+    parser.add_argument('--condition-approved-merger-required', default=None, type=str, help='list of usernames of approved mergers')
```

Comment:
change they way that this accepts a list, it was asking the user enter a python style list ([stuff,here]) but switched to a string that is comma separated and then the arg is handled when used, to split on the commas and create a list to parse.

Other suggestions also welcomed.